app/hooks/use-main-menu-items.tsx

```diff
@@ -29,6 +29,11 @@ export function useMainMenuItems() {
       to: "locations",
       label: "Locations",
     },
+    {
+      icon: <Icon icon="calendar" />,
+      to: "calendar",
+      label: "Calendar",
```

Comment:
We should use different icons for the calendar and bookings. Can you make it look like the icons on the figma file